packages/backend/src/utils/captcha.ts

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios'
+
+let CAPTCHA_SECRET_KEY: string
```

Comment:
what are you going to use this variable for? because I don't see this variable being assigned a value everywhere
In this case, this variable may not be useful

Reply:
oh I forgot to use it lol. the idea is it will be used as a cache so that the `getSecretKey()` doesn't have to check the secret key every time, so only check once. Will fix this one

Reply:
I have a suggestion for you to use some library for caching in nodejs e.g [node-cache](https://www.npmjs.com/package/node-cache) if you want to use it as a cache
personally I dont believe its a good idea to use plain variable to store cached values

Reply:
Hmm, doesn't it seems to be too overkill? I say cache, but basically it is just to store a constant value which is exactly the same from .env variable, and will never change. Do you still want to change it?

Reply:
Plese, do as you see fit 👍

Reply:
Kinda agree that using the let variable is not a good idea, so change it a bit so that I only use const. Should be cleaner now